<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsWindow</class>
 <widget class="QWidget" name="PrefsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences | Randomize Me</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/main/logo.png</normaloff>:/main/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="1">
      <widget class="QPushButton" name="open_repo_btn">
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <property name="text">
        <string>open repository</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/buttons/media/external.png</normaloff>:/buttons/media/external.png</iconset>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;UwU-ify&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;edit lists&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
        </font>
       </property>
       <property name="text">
        <string>wow rly?? :3</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3" rowspan="9">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>preferences</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="about_btn">
       <property name="text">
        <string>about this project</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/buttons/media/external.png</normaloff>:/buttons/media/external.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="open_lists_btn">
       <property name="text">
        <string>lists.py</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QPushButton" name="quit_btn">
       <property name="text">
        <string> close</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/buttons/media/cross.png</normaloff>:/buttons/media/cross.png</iconset>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;updates?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="open_genlog_btn">
       <property name="text">
        <string>sval.txt</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;generation log&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
